### PR TITLE
docs: update peers

### PR DESCRIPTION
**Pull request by alanshaw**

### CODE REVIEW

File: PEERS [hunk 1]
```diff
@@ -7,33 +7,41 @@
...
-# web3-storage-sv15-2
-/ip4/139.178.68.91/tcp/4001/p2p/12D3KooWEDMw7oRqQkdCJbyeqS5mUmWGwTp8JJ2tjCzTkHboF6wK
-# web3-storage-sv15-3
-/ip4/139.178.69.93/tcp/4001/p2p/12D3KooWRi18oHN1j8McxS9RMnuibcTwxu6VCTYHyLNH2R14qhTy
-# web3-storage-sv15-4
-/ip4/139.178.91.227/tcp/4001/p2p/12D3KooWKhPb9tSnCqBswVfC5EPE7iSTXhbF4Ywwz2MKg5UCagbr
-# web3-storage-sv15-5
-/ip4/139.178.91.231/tcp/4001/p2p/12D3KooWAdxvJCV5KXZ6zveTJmnYGrSzAKuLUKZYkZssLk7UKv4i
 # web3-storage-am6
 /ip4/147.75.33.191/tcp/4001/p2p/12D3KooWPySxxWQjBgX9Jp6uAHQfVmdq8HG1gVvS1fRawHNSrmqW
```
Comment (vasco-santos) [inline PEERS:11]:
```suggestion
```

Comment (vasco-santos) [reply PEERS:11]: This was removed right?

**Review by vasco-santos — APPROVED** [review]

File: PEERS (hunk 1 above)
Comment (vasco-santos) [reply PEERS:11]: Forget, I see not

**Comment by thattommyhall** [comment]
```
 "web3_storage_cluster": [
    {
      "ID": "12D3KooWPySxxWQjBgX9Jp6uAHQfVmdq8HG1gVvS1fRawHNSrmqW",
      "Addrs": [
        "/dns/web3-storage-am6.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWSnniGsyAF663gvHdqhyfJMCjWJv54cGSzcPiEMAfanvU",
      "Addrs": [
        "/dns/web3-storage-dc13.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWR19qPPiZH4khepNjS3CLXiB7AbrbAD4ZcDjN1UjGUNE1",
      "Addrs": [
        "/dns/web3-storage-sv15.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWNuoVEfVLJvU3jWY2zLYjGUaathsecwT19jhByjnbQvkj",
      "Addrs": [
        "/dns/web3-storage-am6-2.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-2.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWKytRAd2ujxhGzaLHKJuje8sVrHXvjGNvHXovpar5KaKQ",
      "Addrs": [
        "/dns/web3-storage-dc13-2.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-2.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWEDMw7oRqQkdCJbyeqS5mUmWGwTp8JJ2tjCzTkHboF6wK",
      "Addrs": [
        "/dns/web3-storage-sv15-2.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-2.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWQYBPcvxFnnWzPGEx6JuBnrbF1FZq4jTahczuG2teEk1m",
      "Addrs": [
        "/dns/web3-storage-am6-3.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-3.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWJEfH2MB4RsUoaJPogDPRWbFTi8iehsxsqrQpiJwFNDrP",
      "Addrs": [
        "/dns/web3-storage-dc13-3.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-3.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWRi18oHN1j8McxS9RMnuibcTwxu6VCTYHyLNH2R14qhTy",
      "Addrs": [
        "/dns/web3-storage-sv15-3.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-3.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWDdzN3snjaMJEH9zuq3tjKUFpYHeSGNkiAreF6dQSbCiL",
      "Addrs": [
        "/dns/web3-storage-am6-4.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-4.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWHpE5KiQTkqbn8KbU88ZxwJxYJFaqP4mp9Z9bhNPhym9V",
      "Addrs": [
        "/dns/web3-storage-dc13-4.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-4.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWKhPb9tSnCqBswVfC5EPE7iSTXhbF4Ywwz2MKg5UCagbr",
      "Addrs": [
        "/dns/web3-storage-sv15-4.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-4.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWEzCun34s9qpYEnKkG6epx2Ts9oVGRGnzCvM2s2edioLA",
      "Addrs": [
        "/dns/web3-storage-am6-5.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-5.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWBHvsSSKHeragACma3HUodK5FcPUpXccLu2vHooNsDf9k",
      "Addrs": [
        "/dns/web3-storage-dc13-5.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-5.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWAdxvJCV5KXZ6zveTJmnYGrSzAKuLUKZYkZssLk7UKv4i",
      "Addrs": [
        "/dns/web3-storage-sv15-5.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-5.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWQE3CWA3MJ1YhrYNP8EE3JErGbrCtpKRkFrWgi45nYAMn",
      "Addrs": [
        "/dns/web3-storage-am6-6.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-6.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWMaTJKNwQJyP1fw3ftGb5uqqM2U24Kam8aWqMRXzWHNiF",
      "Addrs": [
        "/dns/web3-storage-dc13-6.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-6.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWRgXWwnZQJgdW1GHW7hJ5UvZ8MLp7HBCSWS596PypAs8M",
      "Addrs": [
        "/dns/web3-storage-sv15-6.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-6.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWDYVuVFGb9Yj6Gi9jWwSLqdnzZgqJg1a1scQMDc4R6RUJ",
      "Addrs": [
        "/dns/web3-storage-am6-7.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-7.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWNCmYvqPbeXmNC4rnTr7hbuVtJKDNpL1vvNz6mq9Sr2Xf",
      "Addrs": [
        "/dns/web3-storage-dc13-7.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-7.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWPbxiW4wFYHs7MwCQNqK9YVedH7QYZXJKMFVduhwR1Lcs",
      "Addrs": [
        "/dns/web3-storage-sv15-7.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-7.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWSafoW6yrSL7waghFAaiCqGy5mdjpQx4jn4CRNqbG7eqG",
      "Addrs": [
        "/dns/web3-storage-am6-8.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-am6-8.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWDRak1XzURGh9MvGR4EWaP9kcbmdoagAcGMcNxBXXLzTF",
      "Addrs": [
        "/dns/web3-storage-dc13-8.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-dc13-8.web3.dwebops.net/tcp/4001"
      ]
    },
    {
      "ID": "12D3KooWSH5uLrYe7XSFpmnQj1NCsoiGeKSRCV7T5xijpX2Po2aT",
      "Addrs": [
        "/dns/web3-storage-sv15-8.web3.dwebops.net/udp/4001/quic",
        "/dns/web3-storage-sv15-8.web3.dwebops.net/tcp/4001"
      ]
    }
  ],
```

is the machine I think should be peered (and how we add them to our own infra)

File: PEERS (hunk 1 above)
Comment (thattommyhall) [reply PEERS:11]: Some of the machines had cluster disabled, but still have an IPFS node running.

**Comment by alanshaw** [comment]
Note, I have a [script that lists these out](https://github.com/alanshaw/dotStorage-tools/blob/0a4185db793d8ab0dbff9e49c7e1d7cea00f4bde/index.js#L14-L38) like this by calling `/api/peers` so if there's any error here it is probably a misconfiguration of the cluster.

**Comment by thattommyhall** [comment]
Whats active in cluster vs whats worth peering with are different though. Current status is there are 5 nodes whos cluster ID has been moved to new hardware, they are currently being pinned but until that is complete the sunsetting nodes (that have cluster stopped, but IPFS running) are worth peering with

**Pull request merged**